### ISSUE TITLE
The Code and Accessibility tabs are missing in the Web Components Storybook.

### ISSUE THREAD

**Issue by anamikaanu96** [carbon-design-system/ibm-products#7383]
Add Code tab and accessibility tab.

<img width="1726" alt="Image" src="https://github.com/user-attachments/assets/44842f85-f0dd-4dd8-96fb-c29aa3d2b0aa" />
